web/src/main/kotlin/no/nav/su/se/bakover/web/routes/vedtak/VedtakJson.kt

```diff
@@ -106,3 +107,20 @@ internal fun Vedtak.IngenEndringIYtelse.toJson(): VedtakJson = VedtakJson(
     periode = periode.toJson(),
     type = vedtakType,
 )
+
+internal fun VedtakSomKanRevurderes.toJson(): VedtakJson = VedtakJson(
```

Comment:
VedtakSomKanRevurderes kan være EndringIYtelse og IngenEndringIYtelse og disse to har egne toJson()-funksjoner. Ville det gitt mening og gjenbruke de? Hvis ikke synes jeg kanskje vi skal ha en egen json-type for denne som ikke kan ha nullables.

Reply:
Landet vel på at måten disse dataene brukes på ikke nødvendigvis tilsier et _behov_ for å bruke de egne json-typene, tok vel en liten snarvei her. Kan fikse.